### PR TITLE
hack browser-pack

### PR DESCRIPTION
**Pull request by heavyk**
hey dude, just saw this repo. I do a similar thing to pack our server installs, but I like your implementation more. it's cleaner.

so I ported my technique to get browser-pack to cooperate.

instead of making my own browser-pack, I just modify the one in my node_modules directory and provide a custom prelude.js. works great! thought I'd share :)

cheers

### CODE REVIEW

**Comment by dominictarr** [comment]
wow, okay it's quite dangerous to update the code of a dependency like you are doing here.
If browser-pack was updated then it might break your code, and if someone was trying to debug browser-pack and this change had been made, it would not be obvious at all. I have been doing node for years and this is the first time I have ever seen any one update the code of a dependency like this.
So, if I was trying to debug this, the possibility that the dependency code was changing would be the last thing that I would expect! When I eventually figured it out, possibly after several days banging head against the wall, I would understandably be very upset with you.

Basically, once a module is installed (& compiled if necessary), it's code should never change, and neither should any state be stored in the `node_modules` folder.

It would be better to just have a fork of browser-pack, or even better get a pull request merged.
